lib/cli.rb

```diff
@@ -137,6 +137,28 @@ def run
       end
     end
 
+    command :use do |c|
+      c.syntax = 'cry use <team/folder>'
+      c.description = 'Select the current folder'
+
+      c.action do |args|
+        TTY::Exit.exit_with(:usage_error, 'Arguments missing') unless args.length >= 1
```

Comment:
maybe move argument check to own method?

team_name, folder_name = use_args

def use_args(args) ...

generic error message? just return usage?